deployment/terraform/task-definitions/app_cli.json

```diff
@@ -2,9 +2,7 @@
   {
     "name": "django",
     "image": "${image}",
-    "cpu": ${cpu},
-    "memory": ${memory},
-    "essential": true,
```

Comment:
This is the only container in the task, so it doesn't matter if we mark it as `essential` or not.